tests/ci/cdk/cdk/ssm/general_test_run_ssm_document.yaml

```diff
@@ -14,31 +14,55 @@ mainSteps:
         # Codebuild just "stops" the instance calling the script, so "trap cleanup" is not executed.
         - shutdown -P +60
         - sudo -i
+        - systemctl stop apt-daily.timer
         - export DEBIAN_FRONTEND=noninteractive
         - export CPU_TYPE=$(dpkg --print-architecture)
+        - export SOURCE={SOURCE}
         # if we have a cpu type of x86, we want linux-x86
         - if [ "${CPU_TYPE}" = amd64 ]; then export CPU_ARCH=linux-x86; export AWS_CLI_PREFIX=x86_; sudo sh -c "echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo"; fi
```

Comment:
Why prevent the turboing? Also does that even work if it's not a metal instance?